ml_metadata/tools/mlmd_bench/README.md

```diff
@@ -1,11 +1,29 @@
 # mlmd_bench
 
-*mlmd_bench* is a MLMD benchmark tool that can measure the MLMD query facility
-performance and scalability for different backends and deployment settings.
-The performance metrics of interests include throughputs of concurrent
-operations and data sizes.
+mlmd_bench is a MLMD benchmark tool that can measure the MLMD query facility performance and scalability for different backends and deployment settings. The performance metrics of interests include throughputs of concurrent operations and data sizes.
 
-## HOW TO USE
+## Introduction
+
+MLMD exposes a provenance graph data model, which consists of {Artifact, Execution, Context, Type} as nodes, and {Event, Association, Attribution, Instance} as edges. Based on the data model, it defines a set of APIs, such as creating and updating nodes and edges, listing nodes by types, traversing nodes via edges.  
+
+The APIs are implemented on various backends (e.g., MySQL, Sqlite, CNS, Spanner) and deployed in different modes (e.g., library, grpc server). On the other hand, the integration of MLMD with different partners have different use cases and exhibit different read/write workloads. To improve API performance, it often requires proper optimization in MLMD query implementation or schema reorganization.    
+
+To guide the performance tuning of MLMD and better support users, we build a benchmarking tool, mlmd_bench, which can:
```

Comment:
To better support users and guide the performance tuning of MLMD

Reply:
Good point. Done.